tetris.py

```diff
@@ -113,6 +118,7 @@ def __init__(self, canvas, width, height, outline_color, current_block_center, s
         self.small_board = small_board
         self.draw_board()
         self.draw_block()
+        self.game_status = game_status
```

Comment:
No need to take it as argument:

```suggestion
        self.game_status = GameStatus.in_progress
```

Reply:
Renamed to ENUM_game_status

Reply:
Removed